packages/data/src/onboarding/actions.js

```diff
@@ -84,3 +186,67 @@ export function* updateProfileItems( items ) {
 		throw new Error();
 	}
 }
+
+export function* snoozeTask( id ) {
+	yield snoozeTaskRequest( id );
+
+	try {
+		yield apiFetch( {
+			path: `${ WC_ADMIN_NAMESPACE }/onboarding/tasks/${ id }/snooze`,
+			method: 'POST',
+		} );
+
+		yield snoozeTaskSuccess();
```

Comment:
Similar as above, we should pass the returned `task` into the success action, so we can also set `snoozedUntil`